Dietpi-Software | Fail2Ban: Fix drop-in config directory pre-creation

Joulinar opened the pull request:
**Status**: Ready | 
- [x] update `dietpi-software`
- [x] tested on RPI3B+

**Reference**: on current installation you will see an error messages passing by

```
[ SUB1 ] DietPi-Software > Installing Fail2Ban: prevents brute-force attacks with ip ban
/boot/dietpi/dietpi-software: line 4529: /etc/fail2ban/fail2ban.d/97_dietpi.conf: No such file or directory
[ INFO ] DietPi-Software | APT install for: python3-systemd fail2ban, please wait...
```

**Commit list/description**:
- DietPi-Software | change order of activities. Install Fail2Ban first before creating any config files

Joulinar committed:
Update dietpi-software

change order of activities. Install Fail2Ban first before creating any config files

MichaIng commented:
Ah, the idea was actually to create the config file first, so there will never be a log file created. Hmm, at least it does not break something 😄.

I'll change it so that the directory is pre-created, so the config file can be as well.

**EDIT: Ah, it was, but a wrong directory. It is `fail2ban.d`, not `fail2ban.conf.d`.**

MichaIng committed:
v6.34

+ DietPi-Software | Fail2Ban: Create config file before installing package, to prevent any log file creation. But fix wrong drop-in config dir pre-creation.
